src/main/java/net/redstoneore/legacyfactions/cmd/CmdFactionsColeader.java

```diff
@@ -0,0 +1,84 @@
+package net.redstoneore.legacyfactions.cmd;
+
+import mkremins.fanciful.FancyMessage;
+import net.redstoneore.legacyfactions.Permission;
+import net.redstoneore.legacyfactions.Role;
+import net.redstoneore.legacyfactions.Lang;
+import net.redstoneore.legacyfactions.entity.Conf;
+import net.redstoneore.legacyfactions.entity.FPlayer;
+import net.redstoneore.legacyfactions.entity.Faction;
+
+import org.bukkit.ChatColor;
+
+public class CmdFactionsColeader extends FCommand {
```

Comment:
💡 We should start using javadocs on classes like this, as it may be accessed as part of the API